src/components/Login/LoginMobileNumberCard.tsx

```diff
@@ -61,7 +61,8 @@ export const LoginMobileNumberCard: FunctionComponent<LoginMobileNumberCard> = (
   };
 
   const onChangeMobileNumber = (text: string): void => {
-    /^\d*$/.test(text) && setMobileNumberValue(text);
+    ///^\d*$/.test(text) &&
```

Comment:
Do be mindful of commented out code.

In this case, as the above comment, we shouldnt need to make changes to `LoginMobileNumberCard` in this PR